src/lib/get-api-version.js

```diff
@@ -12,7 +14,11 @@ const getApiVersion = async () => {
     version = ddoc.deploy_info && ddoc.deploy_info.version;
   }
 
-  return version;
+  return semver.valid(version) ? version : null;
+};
+
+const getValidApiVersion = async () => {
+  return semver.valid(await getApiVersion());
```

Comment:
I'm a bit confused, it's doing `semver.valid` twice, is that intentional? 
Maybe make `getApiVersion()` return `version` as it is, then `getValidApiVersion` can do the validation. 
I think `semver.valid` already returns null, if the version format is wrong.

Reply:
:facepalm:   Whoops!  Yes, I meant to do exactly what you said. After I added the new `getValidApiVersion` function I forgot to go back and update `getApiVersion`...